L2 Understading electric vehicless/Identifying core and secondary parts of a Battery Electric Vehicle (BEV).md

```diff
@@ -0,0 +1,125 @@
+
+Looking at Image 3, we see the complete powertrain of an electric vehicle. 
+
+1. Power electronic controller - *Shown in Image 4*
+2. Stator (Part of a motor) - *Shown in Image 5*
+3. Rotor (Part of a motor) - *Shown in Image 5*
+4. Single-speed gearbox and differential.
+
+# Power electronic controller 
+
+-=- ![Renault-Zoe-power-electronics.jpg](https://demo.pflms.com/markdown_attachments/1727/MYLSElbERkIx65752XQlyg)*Image 4 - Renault Zoe power electronics
+Source: [Renault Cleon Plant press kit](https://www.autoblog.com/photos/renault-r250-electric-motor-at-cleon-plant/#slide-1448570)* 
+
+Looking at Image 4, we see an example Power electronic controller of an electric vehicle. 
+
+Here are the highlighted parts: 
+1. Rectifier
+2. DC-DC converter
+3. Input filter
+4. Inverter
+
+ The power electronics control module has several subsystems, each subsystem is responsible for a control function. 
+
+When the vehicle is charged from a home electrical grid (e.g. 220 V),
+
+1. The **rectifier** converts the alternating current (AC) into direct current (DC), which is fed into the high voltage battery. 
+
+2. The **DC-DC converter** is responsible with the lowering of the high voltage (e.g. 400 V) to the low voltage network (12 V).
+
+3. The **inverter** controls the electric machine speed and torque by converting the direct current from the battery into the alternating 3-phase current for the electric machine. 
+
+4. When the vehicle is in the energy recuperation phase (braking) the inverter is doing the opposite conversion, from 3-phase AC to DC.
+
+5. The **input filter** reduces the electric noise or interference generated during regenerative braking, when energy is fed back into the batteries via the motors.
+
+# Understanding motors
+
+<img class="mx-auto w-auto md:w-auto" alt="A typical exploded view of an Induction motor" src="https://do7js0tdxrds1.cloudfront.net/6cl615vw8or1epho8kcd62d3asnm?response-content-disposition=inline%3B+filename%3D%22AC+%282%29.png%22%3B&response-content-type=image%2Fpng&Expires=1693398434&Signature=Hof2v86sr1eKcBxJkHWLXxfJzREug3WCzphtv6fdXmZpZfihvCxWWqiolfQjzHlpMeR8X5Cw~rhvU9h1b5hGe7hXCANwQKcgz4tdrmvIKEI36s3cpa-fSZtUVyBzw4u2WOFVMCeiDy54eel1SJlnHaGPeOIsQ~LUUtUtD0s90MR2g3jEfPpPQ2J49r2LuvU0afXT8wzusCo8ueqmNzOc9gGC9TCkY~4vG7bYJfz7QMXnLC2qDEpNGw7DN73iShyI43EW6~8bPsH3FvGeH-GLh3qjNHhXGO5tceO9wfF92MX4RPq5Q97ahLX-DjUFCfmtz5Gk8S~AzxbRIf70hOGgpA__&Key-Pair-Id=K2Q3HDJ6ZAQGFF">
+
+**In our upcoming lesson, we will delve deeper into the subject of electric motors.**
+
+The torque is provided by an electric machine. In passenger vehicle applications, there are mainly two types of electric motors already in use, 
+
+- Permanent magnet machines.
+- Inductance machines.
+
+It’s more appropriate to call them electric machines instead of motors because they can also generate electrical energy during vehicle braking. This mechanism is called energy recuperation/regeneration.
```

Comment:
```suggestion
It’s more appropriate to call them electric machines instead of motors because they can also generate electrical energy during vehicle braking. This mechanism is called _energy recuperation/regeneration_.
```